### PR TITLE
Doc page for Card component

### PR DESCRIPTION
**Pull request by GomezIvann**

### CODE REVIEW

File: website/pages/components/card/specifications.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import Head from "next/head";
+import type { ReactElement } from "react";
+import CardSpecsPage from "../../../screens/components/card/specs/CardSpecsPage";
+import CardPageLayout from "../../../screens/components/card/CardPageLayout";
+
+const Specifications = () => {
+  return (
+    <>
+      <Head>
+        <title>Card — Halstack Design System</title>
```
Comment (marcialfps) [inline website/pages/components/card/specifications.tsx:10]: Change `Card` to `Card Specs`.

File: website/screens/components/card/usage/CardUsagePage.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            that presents the information in a clear way and apply the same
+            styles for every card.
+          </DxcText>
+          <DxcText>
+            If a collection want be create, won't use different card styles,
+            use the same to keep consistency.
+          </DxcText>
+        </DxcList>
+      </DxcStack>
+      <DocFooter githubLink="https://github.com/dxc-technology/halstack-style-guide/blob/master/website/screens/components/spinner/usage/SpinnerUsagePage.tsx" />
```
Comment (marcialfps) [inline website/screens/components/card/usage/CardUsagePage.tsx:22]: Github link is not correct.

File: website/screens/components/card/specs/CardSpecsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+            <tr>
+              <td>
+                <Code>xxlarge</Code>
+              </td>
+              <td>100px</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+      </DxcStack>
+      <DocFooter githubLink="https://github.com/dxc-technology/halstack-style-guide/blob/master/website/screens/components/spinner/usage/SpinnerUsagePage.tsx" />
```
Comment (marcialfps) [inline website/screens/components/card/specs/CardSpecsPage.tsx:355]: Github link is not correct.

**Review by marcialfps — CHANGES_REQUESTED** [review]

File: website/pages/components/card/specifications.tsx (hunk 1 above)
Comment (GomezIvann) [reply website/pages/components/card/specifications.tsx:10]: done!

File: website/screens/components/card/specs/CardSpecsPage.tsx (hunk 3 above)
Comment (GomezIvann) [reply website/screens/components/card/specs/CardSpecsPage.tsx:355]: done!

File: website/screens/components/card/usage/CardUsagePage.tsx (hunk 2 above)
Comment (GomezIvann) [reply website/screens/components/card/usage/CardUsagePage.tsx:22]: fixed!

**Review by marcialfps — APPROVED** [review]

**Pull request merged**